drivers/video/video_emul_rx.c

```diff
@@ -67,7 +67,9 @@ static int emul_rx_set_fmt(const struct device *const dev, struct video_format *
 	}
 
 	/* Cache the format selected locally to use it for getting the size of the buffer  */
+	fmt->pitch = fmt->width * video_bits_per_pixel(fmt->pixelformat) / BITS_PER_BYTE;
```

Comment:
This is a small semantic change and I proposed a migration-guide entry here to avoid getting in the way of this PR:
- https://github.com/zephyrproject-rtos/zephyr/pull/90110

This is not a breaking change, and a migration guide entry is only a hint for how to improve applications.